src/chademo.cpp

```diff
@@ -64,23 +66,56 @@ void ChaDeMo::SetChargeCurrent(uint8_t current)
       rampedCurReq--;
 }
 
-void ChaDeMo::SendMessages()
+void ChaDeMo::CheckSensorDeviation(uint16_t internalVoltage)
+{
+   int vtgDev = (int)internalVoltage - (int)chargerOutputVoltage;
+
+   vtgDev = ABS(vtgDev);
+
+   if (vtgDev > 10)
+   {
+      vtgTimeout++;
```

Comment:
I'd suggest that a better name might be vtgErrCnt as this isn't a timeout. Same for curTimeout.

Reply:
It's both. The function must be called at 100ms intervals so a counter of 10 corresponds to 1000 ms. The chademo spec specifies a timeout. So I thought that's what I'd call it.

Reply:
Ok, makes sense then. Thanks for the additional context.